src/main/java/com/github/igotyou/FactoryMod/utility/FactoryModGUI.java

```diff
@@ -283,10 +298,11 @@ private InventoryComponent constructFactoryCreationComponent(FurnCraftChestEgg f
 		section.addComponent(inputSection, SlotPredicates.rectangle(6, 4));
 
 		//TODO opens the setup recipe instead if is upgrade TODO TODO TODO
```

Comment:
While you're at it, do you also want to do what this comment and https://github.com/CivClassic/FactoryMod/issues/15 are talking about?